wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack.swift

```diff
@@ -207,7 +220,10 @@ public class CoreDataStack: NSObject, ContextProvider {
         if needsMigration {
             onStartMigration()
         }
-
+        // this activity should prevent app to be killed while migrating db
+        guard let activity = BackgroundActivityFactory.shared.startBackgroundActivity(withName: "database setup") else {
+            return onFailure(CoreDataStackError.noDatabaseActivity)
```

Comment:
```
onFailure(...
return
```

or

```
return onFailure ... ?
```

Reply:
changed, but it's the same since Swift X, don't remember